src/tests/ftest/util/results_utils.py

```diff
@@ -369,3 +481,100 @@ def create_html(job, results):
     from avocado_result_html import HTMLResult
     result_html = HTMLResult()
     result_html.render(sanitize_results(results), job)
+
+
+class LaunchTestName():
+    """Define a launch.py test name compatible with avocado's result render classes."""
+
+    def __init__(self, name, order, repeat):
+        """Initialize a LaunchTestName object.
+
+        Args:
+            name (str): test name
+            order (int): order in which this test is executed
+            repeat (int): repeat count for this test
+        """
+        self.name = name
+        self.order = order
+        self.repeat = repeat
+
+    def __str__(self):
+        """Get the test name as a string.
+
+        Returns:
+            str: combination of the order and name
+        """
+        if self.repeat > 0:
+            return f"{self.uid}-{self.name}{self.variant}"
+        return f"{self.uid}-{self.name}"
+
+    def __getitem__(self, name, default=None):
+        """Get the value of the attribute name.
+
+        Args:
+            name (str): name of the class attribute to get
+            default (object, optional): value to return if name is not defined. Defaults to None.
+
+        Returns:
+            object: the attribute value or default if not defined
+        """
+        return self.get(name, default)
+
+    def get(self, name, default=None):
+        """Get the value of the attribute name.
+
+        Args:
+            name (str): name of the class attribute to get
+            default (object, optional): value to return if name is not defined. Defaults to None.
+
+        Returns:
+            object: the attribute value or default if not defined
+        """
+        try:
+            return getattr(self, name, default)
+        except TypeError:
+            return default
```

Comment:
What is this for? These are basically the same
```
obj.get(name, default)
```
```
getattr(obj, name, default)
```
And when is a `TypeError` raised?

Reply:
This is required to be compatible with avocado's HTMLResult.render() method.